airflow-core/docs/installation/prerequisites.rst

```diff
@@ -24,11 +24,11 @@ Airflow® is tested with:
 
 * Databases:
 
-  * PostgreSQL: 12, 13, 14, 15, 16
+  * PostgreSQL: 13, 14, 15, 16, 17
```

Comment:
Is that correct? i am not sure we should change support for deps in 3.0 ?

Reply:
Yeah, it came from cherry picking: https://github.com/apache/airflow/pull/54343/commits/9769c0d7ec3f25a7a7077776aeff37002936e19f, maybe I drop it?

Reply:
Dropped it in https://github.com/apache/airflow/commit/d96d769ffe789e8c6039959a8d34528a6509eb06